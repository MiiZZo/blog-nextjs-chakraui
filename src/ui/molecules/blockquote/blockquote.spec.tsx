import React from "react";
import Blockquote from "./blockquote";
import { myRender } from "@lib/testing/render";

describe("blockquote component", () => {
  it("should match a shapshopt", () => {
    const { asFragment } = myRender(<Blockquote>My blockquote</Blockquote>);

    expect(asFragment).toMatchSnapshot();
  });
});
