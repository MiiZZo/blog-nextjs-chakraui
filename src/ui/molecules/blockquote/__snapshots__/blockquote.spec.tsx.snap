// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blockquote component should match a shapshopt 1`] = `[Function]`;
